src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java

```diff
@@ -0,0 +1,19 @@
+package se.bjurr.violations.lib.example;
```

Comment:
Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocPackageCheck
Severity: ERROR
File: /home/bjerre/sandbox/jenkinsnode/workspace/test/violations-test/src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java L0
 

Missing package-info.java file.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>